tests/integrations/export_enrolled_keys/export_enrolled_keys_test.go

```diff
@@ -39,10 +47,36 @@ func TestExportEnrolledKeysEsl(t *testing.T) {
 	out, err := utils.ExecWithOutput("sbctl export-enrolled-keys --dir /tmp/exported-esl --format esl")
 	g.Expect(err).ToNot(HaveOccurred(), out)
 
-	out, err = utils.ExecWithOutput("sig-list-to-certs /tmp/exported-esl/db.esl db")
-	g.Expect(err).ToNot(HaveOccurred(), out)
+	eslReader, err := os.Open("/tmp/exported-esl/db.esl")
+	g.Expect(err).ToNot(HaveOccurred())
+	defer eslReader.Close()
 
-	out, err = utils.ExecWithOutput("openssl x509 -in db-0.der -inform DER -text -noout")
-	g.Expect(err).ToNot(HaveOccurred(), out)
-	g.Expect(out).To(MatchRegexp("Issuer: C=Database Key, CN=Database Key"))
+	sl, err := signature.ReadSignatureList(eslReader)
+	g.Expect(err).ToNot(HaveOccurred())
+
+	s := sl.Signatures[0]
+	certificates, err := x509.ParseCertificates(s.Data)
+	g.Expect(err).ToNot(HaveOccurred())
+	g.Expect(certificates[0].Issuer.CommonName).To(Equal("Database Key"))
+}
+
+func findFileByPattern(dirPath string, pattern string) (string, error) {
+	files, err := ioutil.ReadDir(dirPath)
```

Comment:
`os.ReadDir`, the `ioutil` library is deprecated.

Reply:
good catch! My vim-go/coc setup is completely dysfunctional by the build constraints at the top. I haven't found a way to fix that yet. Anyway, fixed.